lib/utilities/paginator.rb

```diff
@@ -0,0 +1,81 @@
+module Utilities
+  module Paginator
+    def paginator(max_page_blocks: 9,
+                  items_per_page: 5,
+                  items: number_of_all_bets,
+                  current_page:)
+      lp = last_page(items, items_per_page)
+      ptd = pages_to_display(current_page.to_i, max_page_blocks.to_i, lp.to_i)
+      render partial: 'shared/paginator',
+             locals: { blocks: ptd,
+                       current_page: current_page,
+                       previous_status: previous_status(current_page),
+                       next_status: next_status(current_page, lp),
+                       last_page: lp }
+    end
+
+    private
+
+    def last_page(number_of_items, items_per_page)
+      (number_of_items.to_f / items_per_page).ceil.to_s
+    end
+
+    def previous_status(current_page)
+      current_page == '1' ? 'page-item disabled' : 'page-item'
+    end
+
+    def next_status(current_page, last_page)
+      current_page == last_page ? 'page-item disabled' : 'page-item'
+    end
+
+    def pages_to_display(current_page, mpb, number_of_pages)
```

Comment:
I like all of this, but I think we can simplify it. 

Id like to see some test cases for this module, then we can start to refactor it to make it just a little nicer to read through